### PR TITLE
Alerting docs: updates to landing page

### PR DESCRIPTION
**Pull request by brendamuir**
Updates to landing pages after changes to set up and advanced config sections

### CODE REVIEW

File: docs/sources/alerting/_index.md [hunk 1]
```diff
@@ -39,13 +35,13 @@ cards:
...
+    - title: Monitor status
       href: ./manage-notifications/
       description: Monitor, respond to, and triage issues within your services.
       height: 24
-    - title: Monitor
-      href: ./monitor/
-      description: Monitor your alerting metrics to ensure you identify potential issues before they become critical.
+    - title: Additional configuration
+      href: ./set-up/
+      description: Use advanced configuration options to further tailor your alerting setup. While not essential for basic alerting, these options can enhance security, scalability, and automation in complex environments.
```
Comment (ppcano) [inline docs/sources/alerting/_index.md:44]:
```suggestion
      description: Use advanced configuration options to further tailor your alerting setup. These options can enhance security, scalability, and automation in complex environments.
```

Comment (ppcano) [reply docs/sources/alerting/_index.md:44]: Suggest to shorten this message to fit within four lines.

Screenshot 2024-11-11 at 11 45 00


Screenshot 2024-11-11 at 11 45 12

**Comment by ppcano** [comment]
@brendamuir can we update the Docs homepage as part of this PR?


Screenshot 2024-11-11 at 11 48 34

**Comment by brendamuir** [comment]
Sure-not sure where that file is? i can see the landing page..not the home page one?

**Review by ppcano — APPROVED** [review]

**Pull request merged**